operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -205,59 +248,216 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
           - type: string
           - allowed values: true or false
         * ldapRoleMapperClientId: If ldapRoleMapperUseRealmRolesMapping is false then this is the client ID to apply the roles to.
-          - default: shasta
-          - type: string
+            - default: shasta
+            - type: string
+      ```
+
+3. Prepare to generate Sealed Secrets.
+   
+   Secrets are stored in customizations.yaml as `SealedSecret` resources 
+   (encrypted secrets), which are deployed by specific charts and decrypted by the
+   Sealed Secrets operator. But first, those Secrets must be seeded, generated, and
+   encrypted.
+
+   1. Mount the PITDATA so that helm charts are available for the re-install (it might already be mounted).
```

Comment:
Above we are telling them to use ${CSM_DISTDIR} so that seems like it should work here for the paths that use `/mnt/pitdata/${CSM_RELEASE}`.
And places that use `/mnt/pitdata/prep/site-init/utils` should use `/root/site-init/utils`.